### PR TITLE
[Feature] CJ-13

### PR DESCRIPTION
**Pull request by cdsandoval**
**Changes:**

- Add jobs list endpoint

**Issue:**

- [Jira Issue](https://codeablejobs.atlassian.net/browse/CJ-13?atlOrigin=eyJpIjoiNjdjZTcwMzNkMjQ5NDhmNmFhMGIyZGMyNWFkMDA5ODQiLCJwIjoiaiJ9)

### CODE REVIEW

File: app/controllers/concerns/response_manager.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+module ResponseManager
+  extend ActiveSupport::Concern
+
+  def make_response(data, status = 200)
```
Comment (byteconde) [inline app/controllers/concerns/response_manager.rb]: Why should it be 200 by default?

File: app/services/jobby_base_service.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+class JobbyBaseService
```
Comment (byteconde) [inline app/services/jobby_base_service.rb]: What will this service provide us with?

File: app/services/master_services/jobs/list.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    class List < JobbyBaseService
+
+      def initialize(meta:)
+        @page = meta[:page]
+        @per_page = meta[:per_page]
+      end
+
+      def perform
+        data = Job.joins(:tags)
+        puts data.inspect
```
Comment (byteconde) [inline app/services/master_services/jobs/list.rb]: Please, delete **puts**

File: app/controllers/concerns/response_manager.rb (hunk 1 above)
Comment (cdsandoval) [reply app/controllers/concerns/response_manager.rb]: This function is going to be use when render data and the validation of the status is going to be in the controller.

File: app/services/jobby_base_service.rb (hunk 2 above)
Comment (cdsandoval) [reply app/services/jobby_base_service.rb]: This file will contains a shared functionality to others services.

**Pull request merged**